src/dialogs/polkit/dialog.ui

```diff
@@ -147,15 +285,18 @@
         <child>
           <object class="GtkButtonBox" id="buttonbox1">
             <property name="visible">True</property>
-            <property name="can_focus">False</property>
-            <property name="layout_style">expand</property>
+            <property name="can-focus">False</property>
+            <property name="layout-style">expand</property>
             <child>
-              <object class="GtkButton" id="button_cancel">
-                <property name="label" translatable="yes">Cancel</property>
+              <object class="GtkButton" id="button_auth">
```

Comment:
Can we flip the buttons back around plz?

Reply:
I don't even remember changing that in the first place :o